test/smoke-fails/atomic-contended/performBuildRun.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+usage() { echo "Usage: $0 [-a <AOMP directory>]" \
+                        " [-c <Compiler path>]" \
+                        " [-f <Compiler flags>]" \
+                        " [-t <Test source file>]" \
+          1>&2; exit 1; }
+
+while getopts ":a:c:f:t:" o; do
+    case "${o}" in
+        a)
+            AOMP=${OPTARG}
+            ;;
+        c)
+            CC=${OPTARG}
+            ;;
+        f)
+            CFLAGS=${OPTARG}
+            ;;
+        t)
+            TESTSRC=${OPTARG}
+            ;;
+        *)
+            usage
+            ;;
+    esac
+done
+
+# Set 'PROJECT_NAME' to the parent directory's name
+PROJECT_NAME=$(basename $(pwd))
+
+# If 'AOMP' was not specified, fallback to user's AOMP directory
+if [[ -z ${AOMP} ]]; then
+  AOMP="/home/$USER/rocm/aomp"
+fi
+
+# If 'TESTSRC' was not specified: exit
+if [[ -z ${TESTSRC} ]]; then
+  echo "ERROR: No source file specified!"
+  exit 2
+fi
+
+# If 'CC' was not specified: use user's clang
+if [[ -z ${CC} ]]; then
+  CC=${AOMP}/bin/clang
+fi
+
+if [ ! -d ${AOMP} ]; then
```

Comment:
IMHO we should move this check up right after the other `AOMP` related check

Reply:
Will do. Makes sense to me.